### PR TITLE
Modifications to the Manubot section

### PR DESCRIPTION
**Pull request by dhimmel**
Work in progress.

This PR will address the following issues:

- [x] https://github.com/greenelab/meta-review/issues/174
- [x] https://github.com/greenelab/meta-review/issues/141
- [x] https://github.com/greenelab/meta-review/issues/128
- [x] https://github.com/greenelab/meta-review/issues/155#issuecomment-475718465
- [x] https://github.com/greenelab/meta-review/issues/104
- [x] https://github.com/greenelab/meta-review/issues/76 (now mentions pandoc-xnos)

@agitter let me know if there are any other loose ends that this PR could address/

### CODE REVIEW

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: This can be a `###` header.  In #183 I also renamed this section to `Manubot features` to help differentiate it from the `Manubot contribution workflow` section.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Should we call this `Exiting Manubot usage` or `Manubot examples`?  I'm not opposed to the current header if you prefer to keep it.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: I like https://github.com/simonvh/gimmemotifs-manuscript as another example of a mature manuscript that is now available on bioRxiv.  However, it is also a "genomics" manuscript so we would need to reclassify it or https://vsmalladi.github.io/tfsee-manuscript/.  Maybe we call GimmeMotifs a "regulatory genomics" paper?

The deep-rules project could be added to the next sentence.  It is a good example of a fully open collaborative project.  How should we classify it?  As a "tutorial", "educational article", or "research tips"?

**Review by agitter — COMMENTED** [review]
If you include #155, this could also close #104.  It can also make sure we settle on a way to refer to `manubot/rootstock` or `Rootstock` and use it consistently elsewhere.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: Note that I link to the GitLab mirror due to https://github.com/isaacs/github/issues/297#issuecomment-478785501 (GitHub does not have a source-code view for markdown files).

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md:263]: This paragraph could also fit in "Contribution Workflow" section.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: I imagine this paragraph may need be rewritten to be more accessible. However, I thought it was necessary to fill in several missing details of the current infrastructure. Feedback please!

**Comment by dhimmel** [comment]
@agitter can you review now. Also should we add the contribution video in this PR or another?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Does `one-or-more` need the punctuation?

`YAML-formatted` may be an example of technical jargon that can be eliminated.  The rest of the sentence tells readers/users what the system does, which is important, but the details of how it is done may be better in the Manubot documentation.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Pandoc and Travis CI aren't introduced until later sections.  We don't want to disrupt the flow of this paragraph, but it may confuse readers who don't know these systems.

We could remove `or other dependencies such as Pandoc and Travis CI` but that would leave out important parts of how a Manubot manuscript works.  We could explain briefly, such as
>or other dependencies such as Pandoc, which converts document formats, and Travis CI, which automatically builds the manuscript

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: The phrasing could be more active:
> to encapsulate a manuscript in a repository and automatically generate outputs.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Git is now capitalized elsewhere

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Git

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: `on the`

`filters` is the technically correct term, but would calling them `Pandoc extensions` may more accessible?

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: Do we need to use a source code view?  The raw GitHub formatting isn't so bad as an alternative.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:138]: This is less relevant than the other Markdown information in these paragraphs.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:140]: Is there a synonym for `reflow` that is more common?

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md:263]: I like it here for the moment.  We could move it if we want after doing a global read through the full manuscript.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: We should introduce the term `Rootstock` here.  Perhaps:

> An example repository known as a Rootstock

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: `must only`

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Maybe `Upgrading manuscripts to incorporate new Manubot features is also`

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Does this need to be `#contribution-workflow`?

Let's leave this as-is and incorporate the video in #139 after merging.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:262]: Should we clarify that this refers to proficiency as a Manubot manuscript maintainer?  I'd argue that a contributor can be proficient and use only Markdown and GitHub.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:263]: Do we want to note where Markdown is used elsewhere?  I'm reminded of your comment https://github.com/greenelab/meta-review/issues/141#issuecomment-462163613.  It is also prevalent in Jupyter and Rmarkdown notebooks.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:278]: I hadn't seen the quality standards example before.  Cool!

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Add `the` before `manubot cite`

**Review by agitter — CHANGES_REQUESTED** [review]
The new paragraphs and structure are a big improvement.  I have several minor suggestions, mostly in the interest of keeping the descriptions accurate but not overly technical.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:262]: Proficiency defined as "a high degree of competence or skill; expertise." I feel that proficient contributors should be able to view the CI logs to diagnose errors and resolve git conflicts (which can be done through the GitHub UI for the record).

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: The problem is that "Pandoc extensions" has an existing usage:

> The behavior of some of the readers and writers can be adjusted by enabling or disabling various extensions. An extension can be enabled by adding +EXTENSION to the format name and disabled by adding -EXTENSION.

Comment (agitter) [reply content/02.main-text.md]: Good point.  We don't want to incorrectly use an existing term.  We could simply say

> For this functionality, Manubot uses `pandoc-xnos`

I noticed this sentence and the next have `relies on` back to back.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md:262]: I see.  That makes sense.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: I was wrong before, it should be `R Markdown` notebooks per https://rmarkdown.rstudio.com/

**Comment by agitter** [comment]
@dhimmel I re-reviewed and resolved all of my old comments that have been addressed.  There are still a few previous questions and one new comment.  Can you please resolve those conversations if you do not think a change is needed or make updates if you do?

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: The choice is between:

+ https://gitlab.com/manubot/rootstock/blob/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md#L
+ https://github.com/manubot/rootstock/raw/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md

The issue with with the raw.githubusercontent.com URL is that you loose the repository context. However, it may be confusing for people to see the GitLab context. I'm now leaning towards switching to the github raw view.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:138]: Do you think that the following part of the paragraph make sense without this sentence? If so, I'm okay with removing it.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:140]: I couldn't find a word that had such precise meaning:

> (in word processing) the action of rearranging text on a page after having changed such features as type size, line length, and spacing.

It seemed like this was the word sources were using when discussing this concept.

**Comment by dhimmel** [comment]
@agitter somehow I overlooked several of your comments. Everything should now be addressed except for the one-sentence-per-line paragraph, which may need further discussion.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: There isn't a perfect solution, and I agree that there is context lost with the raw URL.  I also slightly prefer https://github.com/manubot/rootstock/raw/fea9eb80af4ca9d5a116f6c28a6b740272e3c0b9/content/02.delete-me.md

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md:138]: Upon re-reading, we can keep this as it is.  The first time I read this I was thinking it described how two _spaces_ at the end of a line produce a newline in the rendered document.  That would have been too detailed for this paragraph.  The actual text supports the rest of the paragraph well.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md:140]: I can't think of anything better either.  Let's leave it.  If I come up with an accurate synonym during file proofreading, I can prose it in a new pull request later.

**Review by agitter — APPROVED** [review]

**Comment by agitter** [comment]
Thanks @dhimmel, your latest comments and commit addressed everything.  You can proceed to merge after resolving the new conflict.

**Pull request merged**
